packages/theme/src/lib/theme.ts

```diff
@@ -260,30 +263,25 @@ const theme = createTheme({
   },
 
   typography: {
-    body1: {
-      fontFamily: "Inter",
-      fontSize: "14px",
-      fontStyle: "normal",
-      fontWeight: 600,
-      lineHeight: "20px",
-    },
-
-    body2: {
-      fontFamily: "Inter",
-      fontSize: "18px",
-      fontStyle: "normal",
-      fontWeight: 500,
-      lineHeight: "22px",
-    },
-
     // custom font theme styles
+    // eslint-disable-next-line sort-keys-fix/sort-keys-fix
     emphasized: {
       fontFamily: "DM Serif Text",
       fontStyle: "italic",
       fontWeight: 400,
     },
 
+    externalLinks: {
```

Comment:
This is something I wanted to get on the same page with the design side so we are using a consistent typography across these type of links and designs. We can utilize/update this over in Studio as well when the opportunities come up.

Reply:
No problem, will create the custom `Link` component. My thought was that we would have a set of Typography variants that would align with design to keep consistent with font copy styles, like we do right now with the color palette, so yea basically a global reference stylesheet in a way....